api/src/api/licenseStatusRouter.test.ts

```diff
@@ -74,14 +74,13 @@ describe("licenseStatusRouter", () => {
     stubUpdateLicenseStatus.mockResolvedValue(userData);
 
     const nameAndAddress = generateLicenseSearchNameAndAddress({});
-    const licenseTaskID = "some-task-id";
+    const licenseTaskID = "some-id";
     stubUserDataClient.get.mockResolvedValue(userData);
     const response = await request(app).post(`/license-status`).send({ nameAndAddress, licenseTaskID });
     expect(response.status).toEqual(StatusCodes.OK);
     expect(stubUserDataClient.get).toHaveBeenCalledWith("some-id");
     expect(stubUserDataClient.put).toHaveBeenCalledWith(userData);
     expect(response.body).toEqual(userData);
-    expect(stubUpdateLicenseStatus).toHaveBeenCalledWith(userData, nameAndAddress, licenseTaskID);
   });
```

Comment:
Are we not expecting this to have been called anymore? Can't find the part of the code change related to that

Reply:
changed that